biarchtool.py

```diff
@@ -46,28 +46,70 @@ def __init__(self, project):
         self.rdeps = None
         self.whitelist = {
                 'i586': set([
+                    'bzr',
+                    'git',
+                    # _link to baselibs package
+                    'libjpeg62-turbo',
+                    'mercurial',
+                    'subversion',
                     'ovmf'
                     ]),
                 }
+        self.blacklist = {
+                'i586': set([
+                    'belle-sip',
+                    'release-notes-openSUSE',
+                    'openSUSE-EULAs', # translate-toolkit
+                    'skelcd-openSUSE',
+                    'plasma5-workspace',
+                    ]),
+                }
+
+
+    def _has_baselibs_in_files(self, files):
+        for n in files.findall("./entry[@name='baselibs.conf']"):
+            return True
+        return False
+
+    def _is_link(self, files):
+        for n in files.findall("./entry[@name='_link']"):
+            return True
+        return False
```

Comment:
Perhaps:
```
return len(files.findall("./entry[@name='_link']")) > 0
```
or
```
return files.find("./entry[@name='_link']") is not None
```
to avoid `findall()`.